test/elixir_search_extractor/file_upload_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule ElixirSearchExtractor.FileUploadTest do
+  use ElixirSearchExtractor.DataCase
+
+  import ElixirSearchExtractor.KeywordsFixtures
+  import ElixirSearchExtractor.AccountsFixtures
+  alias ElixirSearchExtractor.FileUpload
+
+  describe "keyword_files" do
```

Comment:
Let's restructure this test to follow our convention to put function name in `describe` block and description in `test` block.  💭

You can check it here: https://nimblehq.co/compass/development/code-conventions/ex-unit/#formatting

Reply:
Fixed in [504080d](https://github.com/rafayet-monon/elixir_search_extractor/pull/35/commits/504080db30020d341f9ed6cd720a5bf96c1ac143)